chia/protocols/shared_protocol.py

```diff
@@ -17,6 +17,7 @@
 # These are passed in as uint16 into the Handshake
 class Capability(IntEnum):
     BASE = 1  # Base capability just means it supports the chia protocol at mainnet
+    NONERESPONSE = 2
```

Comment:
nitpick: NONE_RESPONSE seems more readable, but please keep as-is if it's for consistency